addon/templates/components/nypr-accounts/membership-card.hbs

```diff
@@ -3,14 +3,16 @@
 {{#m.panel to=1 classNames="nypr-membership-info" title="My Donation Status" as |card|}}
   {{card.header title="My Donation Status"}}
 
-  {{#unless pledgeManagerEnabled}}
-    <p class="membership-redirect-message">{{pledgeRedirectMessage}}</p>
+  {{#if customStatusMessage}}
+    <p class="membership-custom-message">{{{customStatusMessage}}}</p>
+  {{/if}}
+  {{#if customStatusLinkUrl}}
     <a
-      class="nypr membership-redirect-link"
-      href={{pledgeRedirectUrl}}
+      class="nypr membership-custom-link"
+      href={{customStatusLinkUrl}}
       target="_blank" rel="noopener"
-      aria-label={{concat pledgeRedirectUrlText ' (Opens in new window)' }} >{{pledgeRedirectUrlText}}</a>
-  {{/unless}}
+      aria-label={{concat customStatusLinkText ' (Opens in new window)' }} >{{customStatusLinkText}}</a>
```

Comment:
could these use template strings instead of `concat` (I've never seen `concat` like this before, so I might very well not be understanding this)?

Reply:
You can't really use javascript within Ember's handlebars templates. `concat` is a built in handlebars template helper.

An alternative here would be to define this as a computed property in the component controller (membership-card.js), with a new name (maybe customStatusLinkScreenReaderText)

Reply:
gotcha, makes sense.